frontend/public/data/possibleFilters.js

```diff
@@ -34,67 +44,68 @@ const getMembersFilters = filterChoices => [
     type: "openMultiSelectDialogButton",
     key: "skills",
     itemType: "skills",
-    itemsToChooseFrom: filterChoices.skills.map(s => ({ ...s, key: s.id }))
-  }
+    itemsToChooseFrom: filterChoices.skills.map((s) => ({ ...s, key: s.id })),
+  },
 ];
 
-const getOrganizationsFilters = filterChoices => [
+const getOrganizationsFilters = (filterChoices) => [
   {
     icon: LocationOnOutlinedIcon,
     iconName: "LocationOnOutlinedIcon",
     title: "Country",
     type: "select",
     key: "country",
-    options: countries.map(c => ({ key: c.toLowerCase(), name: c }))
+    options: countries.map((c) => ({ key: c.toLowerCase(), name: c })),
   },
   {
     icon: LocationOnOutlinedIcon,
     iconName: "LocationOnOutlinedIcon",
     title: "City",
     type: "text",
-    key: "city"
+    key: "city",
   },
   {
     icon: GroupIcon,
     iconName: "GroupIcon",
     title: "Organization type",
     type: "multiselect",
-    options: filterChoices.organization_types.map(t => ({ ...t, key: t.id })),
-    key: "organization_type"
-  }
+    options: (filterChoices || {}).organization_types.map((t) => ({ ...t, key: t.id })),
+    key: "organization_type",
+  },
 ];
 
-const getProjectsFilters = filterChoices => [
+const getProjectsFilters = (filterChoices) => [
   {
     icon: LocationOnOutlinedIcon,
     iconName: "LocationOnOutlinedIcon",
     title: "Country",
     type: "select",
     key: "country",
-    options: countries.map(c => ({ key: c.toLowerCase(), name: c }))
+    options: countries.map((c) => ({ key: c.toLowerCase(), name: c })),
   },
   {
     icon: LocationOnOutlinedIcon,
     iconName: "LocationOnOutlinedIcon",
     title: "City",
     type: "text",
-    key: "city"
+    key: "city",
   },
   {
     icon: DoneAllOutlinedIcon,
     iconName: "DoneAllOutlinedIcon",
     title: "Status",
     type: "multiselect",
-    options: filterChoices.project_statuses.map(s => ({ ...s, key: s.id })),
-    key: "status"
+    options: filterChoices?.project_statuses.map((s) => ({ ...s, key: s.id })),
+    key: "status",
   },
   {
     icon: GroupIcon,
     iconName: "GroupIcon",
-    title: "Organization type",
+    // A hack: need an extra space character to create some horizontal space between the icon and text
+    title: " Organization type",
     type: "multiselect",
-    options: filterChoices.organization_types.map(t => ({ ...t, key: t.id })),
-    key: "organization_type"
+    options: filterChoices?.organization_types?.map((t) => ({ ...t, key: t.id })),
```

Comment:
Added optional chaining to these ☝️

Reply:
This is great, I didn't know you could do it like that! I was always manually validating each reference in the chain :D

Reply:
Hands down my newish favorite JS feature: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining